packages/block-serialization-default-parser/README.md

```diff
@@ -0,0 +1,107 @@
+# Block Serialization Default Parser
+
+This library contains the default block serialization parser implementations for
```

Comment:
this makes me want to cry since it's something I love about markdown and consistent among every other markdown parser I've used.

nonetheless, I have destroyed my markdown to make it happy in ee72314cc

🙃